src/jdk.jartool/share/classes/sun/tools/jar/resources/jar.properties

```diff
@@ -316,8 +318,11 @@ main.help.opt.extract=\
 \ Operation modifiers valid only in extract mode:\n
 main.help.opt.extract.keep-old-files=\
 \  -k, --keep-old-files       Do not overwrite existing files.\n\
-\                             In particular, if a file appears more than once in an\n\
-\                             archive, later copies will not overwrite earlier copies.
+\                             If a file with the same name exists in the target\n\
+\                             directory, skip the extraction of that file.\n\
```

Comment:
This is OK, but I would probably just  the if the file exists, it will not be overwritten